client/src/components/TextInputComponent/TextInput.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import TextInput from "./TextInput";
+
+storiesOf("TextInput", module)
```

Comment:
You want to define a function here (maybe call it `handleChange`) and then render your `TextInput` component like this:
```jsx
<TextInput type="text" value="" placeholder="Sign Up Here" onChange={handleChange} />
```